src/app/components/auth-page/auth-page.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+
+import {AuthorizationService, RoutingService, UserService} from '../../services';
+
+@Component({
+  selector: 'app-auth-page',
+  templateUrl: './auth-page.component.html',
+  styleUrls: ['./auth-page.component.scss']
+})
+export class AuthPageComponent {
+
+  public loginForm = this.formBuild.group({
+    loginInput: ['', Validators.required],
+    passwordInput: ['', Validators.required],
+  });
+
+  constructor(private authorizationService: AuthorizationService,
+              private formBuild: FormBuilder,
+              private routingService: RoutingService,
+              private userService: UserService) {};
+
+  public onLogin() {
+    let username = this.loginForm.controls['loginInput'].value;
+    let password = this.loginForm.controls['passwordInput'].value;
+
+    if(this.loginForm.invalid) {
+      return;
+    } else {
```

Comment:
else block can be skipped as you already did return

Reply:
done